core/src/test/java/com/graphhopper/routing/ch/CHTurnCostTest.java

```diff
@@ -181,14 +181,14 @@ public void testFindPath_multipleInOutEdges_turnReplacementDifference() {
         setRandomCost(2, 5, 6, rnd);
         setRandomCost(4, 7, 10, rnd);
         setRandomCost(6, 7, 10, rnd);
-        setRandomCost(0, 5, 3, rnd);
-        setRandomCost(1, 5, 3, rnd);
-        setRandomCost(0, 5, 6, rnd);
-        setRandomCost(1, 5, 6, rnd);
-        setRandomCost(4, 7, 8, rnd);
-        setRandomCost(4, 7, 9, rnd);
-        setRandomCost(6, 7, 8, rnd);
-        setRandomCost(6, 7, 9, rnd);
+        setRandomCostOrRestriction(0, 5, 3, rnd);
```

Comment:
just using setRandomCost would not lead to strict restrictions

Reply:
Ups, thanks for the fix!